### DISCUSSION TITLE
Any update on the Matrix Space?

### DISCUSSION THREAD

**Discussion by Charadon** [pulsar-edit/.github#251] [Discussions Meta]
Hey, it's been a while since we heard anything on the Matrix space. Is it still coming?

**Comment by Daeraxa** [comment]
Not for a while, no. The person who was leading that hasn't been as active recently, I was tempted to pick this up again myself to try a slightly easier implementation of the bridge we would need.

**Reply by Charadon** [reply]
I'd work on it, since it'd be pretty easy, but I don't really have *access* to discord, since they always want my phone number and other personal information. =/
